baseplate/clients/cassandra.py

```diff
@@ -231,20 +258,32 @@ def wait_for_callbacks_result(self: ResponseFuture) -> Any:
     return response_future
 
 
-def _on_execute_complete(_result: Any, span: Span, event: Event) -> None:
+def _on_execute_complete(
+    _result: Any, args: Tuple[Span, float, Tuple[Any, ...]], event: Event
+) -> None:
     # TODO: tag with anything from the result set?
     # TODO: tag with any returned warnings
+    span, start_time, prom_labels = args
     try:
         span.finish()
     finally:
+        REQUEST_TIME.labels(*prom_labels, "true").observe(time.perf_counter() - start_time)
```

Comment:
IMO, this should enforce the labels it wants to be added, not apply an arbitrary set of labels provided by the caller. This will make the code easier to debug. It would also make it much easier to explicitly see what metrics are being produced.

Reply:
Agree, this is hard to reason about in comparison with the labels at the top of the file.

Reply:
The only way I can think of to do that would be to make both `_on_execute_complete` and `_on_execute_failed` class method of `CassandraSessionAdapter` at which point they would just maybe re-compute the labels the same way we currently do them here.

Can you explain to me how it makes things better/easier?

Reply:
I believe this comment was addressed, please confirm